models/todolist.py

```diff
@@ -0,0 +1,90 @@
+import datetime as dt
+from models import base_db_model
+
+
+class ToDoList():
+    def __init__(self, userid: str, td_secs: int, msg: str, discord_message, start_time: dt.datetime = dt.datetime.utcnow(), end_time=None):
+        """ Default constructor """
+        self._userid = userid
+        self._msg = msg  # remaining message text
+        self._discord_message = discord_message  # message object
+        self.message_id = self._discord_message.id
+        self.channel_id = self._discord_message.channel.id
```

Comment:
I would recommend using properties as they are more in line with Python's style of OOP design.